test/transform.jl

```diff
@@ -182,7 +188,23 @@ end
 
     upperinds = [LinearIndices(size(x))[I] for I in CartesianIndices(size(x)) if I[2] > I[1]]
     J = ForwardDiff.jacobian(x->link(dist, x), x)
-    J = J[upperinds, upperinds]
+    J = J[:, upperinds]
```

Comment:
What was this for again? Sorry, we might have discussed this before.

Reply:
Don't think we have : ) . It's because the output of `dist` is an `AbstractVector` now, so the indices of upper triagular elements don't apply anymore. In this test, x is 3x3 matrix, `link(dist, x)` is length 3 vector, the Jacobian is then a 3x9 matrix, and we are keeping all output elements (as they are all relevant now) and only the `upperinds` of the input elements.